app/components/request_form/request_form.css

```diff
@@ -10,6 +10,20 @@
   top: calc(3.5 * var(--spacing-unit));
 }
 
+.RequestForm-messageTextAreaToolbar {
+  width: 100%;
+  border: 2px solid transparent;
+  border-radius: 5px;
+  padding: 6px;
+  background-color: var(--color-gray-light);
+  margin: var(--spacing-unit-s) 0;
+}
+
+.RequestForm-insertPlaceholderButton {
+  background-color: #fff;
+  font-size: 1rem;
+}
+
```

Comment:
```suggestion

.RequestForm-insertPlaceholderButton:focus {
  border-color: var(--color-focus);
  outline: none;
  box-shadow: var(--input-focus-shadow);
}

```

Reply:
Why do we need this? If the focus styles don’t work for some reason, we might want to consider to update them for all of our buttons.

Reply:
Turns out that we don't need this cause we use `	postcss-focus-within` to add the focus styles and we just needing to add a `tabindex='0'` to the `<div>`

Reply:
I agree that the focusing behaviour from the example you reference is nice for big toolbars. As long as our toolbar only has one or two or three elements I think tabbing through is probably fine.

Also in regards to the button focus behaviour, summing up Matts` and Tills' comments, I also think we should update the button focus behaviour.

It's currently defined as

```
.Button:focus {
  outline: none;
  box-shadow: var(--input-focus-shadow);
}
```

in `button.css`. And I would propose updating it with a focus-color border-color to:
```
.Button:focus {
  border-color: var(--color-focus);
  outline: none;
  box-shadow: var(--input-focus-shadow);
}
```

This makes focusing behaviour between input elements and buttons consistent.

Reply:
I'll add a short PR with that.